test/e2e/exists_test.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2021 VMware, Inc.
```

Comment:
will it be better to rename this file as `exists_op_test.go` ?

`exists_test.go` name I believe is not  communicating that we are trying to test exists operation here.

Reply:
Makes sense. Will do.